app/controllers/idv/by_mail/request_letter_controller.rb

```diff
@@ -1,7 +1,7 @@
-module Idv
-  class GpoController < ApplicationController
+module Idv::ByMail
```

Comment:
I missed this in the original PR but I think we should use the nested namespace style, not the abbreviated style, ex:
```ruby
module Idv
  module ByMail
    class RequestLetterController < ApplicationController
```

It's more clear which of the intermediate namespaces is a class vs a module and means the Rails autoloader has to do less guessing (because you can't redefine a class as a module and vice versa)

Reply:
FYSA @matthinz, re: #9136. I'm assuming this isn't a blocker.

Reply:
yeah not a blocker